chore: docs updated ipxe (#35)
latitudesh/terraform-provider-latitudesh: docs/resources/server.md

  site             = data.latitudesh_region.region.slug # You can use the site id or slug
  ssh_keys         = [latitudesh_ssh_key.ssh_key.id]
  user_data        = latitudesh_user_data.user_data.id

>   ipxe_url         = "http://some-host.com/some-script.ipxe"

Reviewer: ```suggestion
  ipxe_url         = "" # URL to a boot.ipxe file. e.g. https://boot.netboot.xyz.
```

- `ssh_keys` (List of Number) List of server SSH key ids
- `user_data` (Number) The id of user data to set on the server

> - `ipxe_url` (String) URL where iPXE script is stored on, necessary for custom image deployments. This attribute is required when operating system iPXE is selected.

Reviewer: ```suggestion
- `ipxe_url` (String) Public URL for the iPXE script. Required for custom image deployments. Learn more: https://docs.latitude.sh/docs/custom-image-ipxe
```
